tests/tests.rs

```diff
@@ -7,8 +7,12 @@ use std::usize;
 fn can_iterate_over_allocated_things() {
     let mut bump = Bump::new();
 
+    #[cfg(not(miri))]
     const MAX: u64 = 131_072;
 
+    #[cfg(miri)]
+    const MAX: u64 = 1024;
```

Comment:
Similarly, can you add a comment here explaining why `MAX` is smaller when running under miri? (I assume we time out / take too long to run tests because running under miri is slow?)